php/cubex-workerman/composer.json

```diff
@@ -0,0 +1,20 @@
+{
+  "require": {
+    "cubex/framework": "~4.20.0",
+    "cubex/workerman": "@stable",
+    "packaged/polyfill-ctype": "@stable",
+    "packaged/polyfill-mbstring": "@stable",
+    "packaged/polyfill-intl": "@stable",
+    "packaged/polyfill-php71": "@stable",
+    "packaged/polyfill-php72": "@stable",
+    "packaged/polyfill-php73": "@stable",
+    "packaged/polyfill-php74": "@stable",
+    "packaged/polyfill-php80": "@stable",
+    "packaged/polyfill-php81": "@stable"
```

Comment:
if I understand each php version requires a polyfier, thus I think we need `"packaged/polyfill-php83` only

Reply:
polyfill-php83 currently doesn't exist. 

The existence of these polyfills is simply to prevent symfony bringing in their polyfills (which are unnecessary) as we currently use their http-foundation. (Hopefully to be removed in the next major version 

Cubex can be run without any polyfill library.

Reply:
sane for 82 version ?